### ISSUE TITLE
Leaderboard

### ISSUE THREAD

**Issue by Jacquelyn-Rudis** [YMCACOLUMBUS/activescience-issues#74]
1) Can up to 25 children be listed per site? Right now it is only 5
2) Can sites' info be averaged and listed with other participating site on the Leaderboard (to run inter-site or association challenges)?

**Comment by TAGraves** [comment]
1. The leaderboard should show every student at the site that has at least one point in the selected mission (it scrolls up and down to show more students). If you have more than 5 students with points for a mission and are only seeing 5, please let me know. (Note that the first grade exploring science mission is counted separately from the 3rd grade exploring science mission).
2. This functionality does not currently exist and is not too easily added; it will certainly not be available in our first production release. The reason for the difficulty is that the tablets will only grab the data for the site(s) they are registered to, for security and performance purposes. We can of course add it on at a later time, and I can create a detailed specification document for the future implementation of it if this is determined to be an essential feature.

**Comment by Jacquelyn-Rudis** [comment]
Thank you! The leaderboard that sites using Active Science currently are accessing has 5, I think that having all students who have earned any points is a great idea. Regarding the site comparison, perhaps that feature would just be best accessed through the back end database rather than on the tablets?

**Comment by TAGraves** [comment]
Yes, putting it in the back-end administration panel is much easier.
